recipes/pangeo-forge-recipes-feedstock/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "pangeo-forge-recipes" %}
+{% set version = "0.6.0" %}
+{% set hash_val = "b8d16616a9d9097c41d45d7acc8543245a710b8b450cb687a5f03d1908414d68" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_val }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.8
+    - aiohttp
+    - black
+    - boto3
+    - cfgrib <0.9.9.0
+    - codecov
+    - dask
+    - distributed
+    - fsspec >=2021.6.0
+    - fsspec-reference-maker
+    - h5netcdf
+    - h5py
+    - hdf5
+    - intake
+    - intake-xarray
+    - lxml
+    - netcdf4
+    - numcodecs
+    - numpy
+    - pandas
+    - pip
+    - prefect
+    - pydap
+    - pytest
+    - pytest-cov
+    - pytest-lazy-fixture
+    - rasterio
+    - requests <2.26,>=2.20
+    - rechunker >=0.4.2
+    - scipy
+    - setuptools
+    - toolz
+    - xarray >=0.18.0
+    - zarr >=2.6.0
+    - pytest-repeat
+    - pytest-timeout
```

Comment:
This list of dependencies should be those required to run `pangeo_forge_recipes`, but it seems like there are extra testing and development dependencies (e.g. `pytest`, `black`, etc.) listed too. My guess is the set of packages listed [here](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/8720b48bfe208319ab80135500192e23351eece9/setup.cfg#L28-L41) is what you'll need to run `pangeo_forge_recipes`

Reply:
Thanks for the review, @jrbourbeau! You're totally right, I pulled deps from our CI environment, rather than the `install_requires` list. Attempted a fix with https://github.com/conda-forge/staged-recipes/pull/16146/commits/a51e4da9eeddaea37030df74931913aa9e6e064a